### DISCUSSION TITLE
Google analytics Integration - GA4

### DISCUSSION THREAD

**Discussion by cleankoding-pradeep** [wxt-dev/wxt#555] [Q&A]
Hi,

Is GA4 Integration possible wth wxt? Please suggest any other ways to do this

**Comment by aklinker1** [comment]
WXT doesn't provide anything special for analytics. Follow Google's official guides:

- MV3: https://developer.chrome.com/docs/extensions/how-to/integrate/google-analytics-4
- MV2: https://developer.chrome.com/docs/extensions/mv2/tutorials/analytics
